Documentation/dynamic-rupture.rst

```diff
@@ -153,3 +153,44 @@ Linear-Slip Weakening Friction
 
 Rate-and-State Friction
 ^^^^^^^^^^^^^^^^^^^^^^^
+
+Slip-rate imposed on a DR boundary condition
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This friction law allows imposing slip-rate on a dynamic rupture boundary (potentially any kinematic models, but the current implementation is limited, see below).
+The FL id for this friction law is 33.
+The potential advantages of this approch, compared to a multi point-sources representation are:
+
+- offset discontinuity at the free-surface accurately accounted for
+- wavefield (including final stress or displacement) not impacted by the discretization in the near-field.
+
+The current implementation allows imposing a slip distribution on the DR Boundary using the same arbitrary smooth-step SR function everywhere on the fault.
+The slip distribution is imposed simultaneously everywhere on the fault, smoothly over a time ``t_0``, where ``t_0`` is a parameter of the ``DynamicRupture`` namelist.
+This is an expensive way of getting the final stress distribution from a given slip distribution.
```

Comment:
Currently yes. I agree that it is not very smart to use seissol for a static calculation.
But it seems that the tools for doing that, that our collaborators at INGV and Umich tried rely on simplifying assumptions (e.g. homogeneous, no topography, if boundary element methods) and it seems that the resulting tractions on the fault are not usable because too noisy. Therefore it is useful to have something expensive that is working, and it might be an advantage that it relies on the exact same mesh.